mne/preprocessing/artifact_detection.py

```diff
@@ -4,12 +4,94 @@
 
 
 import numpy as np
-
 from ..annotations import (Annotations, _annotations_starts_stops)
 from ..transforms import (quat_to_rot, _average_quats, _angle_between_quats,
                           apply_trans, _quat_to_affine)
+from ..filter import filter_data
 from .. import Transform
-from ..utils import (_mask_to_onsets_offsets, logger)
+from ..utils import (_mask_to_onsets_offsets, logger, verbose)
+
+
+@verbose
+def annotate_muscle_zscore(raw, threshold=4, picks=None, min_length_good=.1,
+                           filter_freq=[110, 140], n_jobs=1, verbose=None):
```

Comment:
```suggestion
                           filter_freq=(110, 140), n_jobs=1, verbose=None):
```
don't use mutable types for defaults values in function signatures.

See eg https://docs.python-guide.org/writing/gotchas/

Reply:
Thanks for the info on the link!